malibal/aon/s1/default.nix

```diff
@@ -0,0 +1,71 @@
+{ pkgs, lib, ... }:
+
+{
+  imports = [
+    ../../../common/pc/laptop
+    ../../../common/pc/laptop/ssd
+    ../../../common/cpu/intel
+    ../../../common/gpu/intel
+    ../../../common/gpu/nvidia/disable.nix
+  ];
+
+  hardware = {
+    enableAllFirmware = lib.mkDefault true;
+    enableRedistributableFirmware = lib.mkDefault true;
+
+    graphics = {
+      enable = lib.mkDefault true;
+      enable32Bit = lib.mkDefault true;
+	    extraPackages = with pkgs; [
+	    	(if (lib.versionOlder (lib.versions.majorMinor lib.version) "23.11")
+         then vpl-gpu-rt
+         else intel-vaapi-driver)
+	    ];
+    };
+  };
+
+  boot = {
+	  initrd = {
+	    kernelModules = [
+        "i915"  # Early loading so the passphrase prompt appears on external displays
+      ];
+	    availableKernelModules = [
+        "nvme"
+        "xhci_pci"
+        "thunderbolt"
+        "usbhid"
+      ];
+    };
+
+    kernelParams = [
+      "mem_sleep_default=deep"
+      "i915.fastboot=1"
+    ];
+    kernelModules = [
+      "coretemp"
+    ];
+  };
+
+  powerManagement = {
+    powertop.enable = lib.mkDefault true;
+  };
+
+  services = {
+    logind = {
+      extraConfig = ''
+        # Don’t shutdown when power button is short-pressed
+        HandlePowerKey=ignore
```

Comment:
This is the same behavior on other systems. Is there something specific why we disable it here?

Reply:
I have it because it's easy to press the power key as it's right above the backspace key.

I remember that I got the shutdown prompt on KDE when I tapped it, but I can't recall if this is the only thing I changed. I can test this this weekend

Reply:
Ok. If you think this is a an easy to accidental press to make issue, please add a comment why this is the default. Also one concern is that it's not easy to override this way... Maybe leave it comment it out for now and enable it in your own configuration.